### ISSUE TITLE
Find a way to request more memory for a function container

### ISSUE THREAD

**Issue by dsyer** [markfisher/sk8s#137]
The java function invoker has a "-Xmx..." limiting the Java heap, which seems sensible, but probably needs to be parameterized.

**Comment by markfisher** [comment]
possibly related to @ericbottard's plan for #130 since he's now considering expanding that scope to the entire pod template spec for the function container (although that would expose config options related to *limits* not necessarily *allocations* IIUC... maybe the same thing in the `-Xmx...` sense?)

**Comment by ericbottard** [comment]
For the record. I'm aiming for *container* spec, not *pod*, as pod is at a higher level than our sidecar etc

**Comment by markfisher** [comment]
yes, sorry - this is scoped to the function container (not the sidecar). thanks
